README.md

```diff
@@ -128,14 +128,37 @@ Each of the following views should correspond to a color. ex: trip view is yello
 
 
 
-### Tracing Events
 
-**Activity:**
-Use the worksheet that corresponds to all the
-Cut out the squares and put the segments of code in order in which they are ran, based on that event happening
 
-- When Button, 'See All Trips', is clicked
-- When button, 'Make Reservation', is clicked
+## Tracing Events
+
+**Activity**
+It can be difficult to figure out when functions get called when we're listening to events and there's not one line of execution to follow.  Now you and your partner will be diving into the events and the functions listening to those events in this Trek App.  
+
+### DOM Events
+
+Using a whiteboard make a list of the DOM events each of the views are listening to and draw arrows to functions that the events trigger.  
+
+-  When Button, 'See All Trips', is clicked
+-  When button, 'Make Reservation', is clicked
+
+![image of a DOM Event linked to a function](images/diagramEvents.png)
+
+Now look at the event handler functions, do they invoke other events or functions?  Diagram those as well.
+
+### Backbone Events
+
+Look at the listener initialize functions.  Are any of the views listening to events on other objects, such as Models, Collections or Views?
```

Comment:
I think it's saying that they should look at the `initialize` function for each of the listeners for the events they've identified.